### ISSUE TITLE
mr.robot auto checkout not working?

### ISSUE THREAD

**Issue by thet** [plone/buildout.coredev#281]
mr.roboto doesn't add checkouts to checkouts.cfg automatically at the moment.

There were changes in core  packages. The changeset was commited, but not added in checkouts.cfg in 5.1:
https://github.com/plone/buildout.coredev/commit/d79ebcdb1e4e549f75149c2d10d0719c328da0c3
https://github.com/plone/buildout.coredev/commit/aec1cdd13c84469b91f95ce6776a40fe79c3aee4

I had added them manually to checkouts.cfg here:
https://github.com/plone/buildout.coredev/commit/d586c4a41d853752dbfae3c730748419b66146d0

Same for 5.0.

/cc @gforcada @mauritsvanrees

**Comment by gforcada** [comment]
I don't see any error on the log file...  so I'm afraid I can not debug the problem.

@mauritsvanrees as you did some releases lately, did you notice any misbehavior?

**Comment by mauritsvanrees** [comment]
With releases it is the other way around: we remove packages from checkouts. So I don't notice difficulties.

In the case of plone.app.multilingual: I wonder if the merge was done manually on the command line. That may mean nothing is triggered in mr.roboto: direct commits to a branch do not trigger the robots.

**Comment by gforcada** [comment]
@mauritsvanrees good point, @thet was the merge done manually? mr.roboto only listens to github activity and does not check commits directly

**Comment by gforcada** [comment]
@thet if you did not notice any other problem I will close it as a hiccup or as @mauritsvanrees mentions that it was a manual merge, feel free to re-open if you disagree

**Comment by thet** [comment]
Today I had again this problem - I prepared several PRs (summarized here) https://github.com/plone/Products.CMFPlone/pull/1828 - @jensens merged it but nothing was written to checkouts.cfg (Products.CMFPlone and plone.app.upgrades were already present, though).

I added the missing repos by hand:
-    plone.app.widgets
-    Products.Archetypes
-    plone.app.z3cform
-    plone.app.contentrules
https://github.com/plone/buildout.coredev/commit/94931ad438983605c08b00e2bd78b96f7cac3b09

Also, the repos targeting 5.0 were not auto-added to checkouts.cfg there either:
https://github.com/plone/buildout.coredev/commit/e9107fe896e8f14f1afae39ff412dad4b63fab63
here, only plone.app.contentrules was missing and plone.app.upgrades already in place.

maurits was doing released today, so there might be an connection to this.

/cc @gforcada @mauritsvanrees
